src/lib/saveState.js

```diff
@@ -13,6 +13,7 @@ export const SaveState = {
       savedSession: window.store.getState().savedSession,
       settings: window.store.getState().settings,
       version: CURRENT_OPTIMIZER_VERSION,
+      inventoryWidth: window.store.getState().inventoryWidth,
     }
 
```

Comment:
We probably need 2-3 options here to make this usable for large inventories

inventoryWidth: number
filteredSlot: enabled by default
filteredSet: disabled by default

otherwise its going to be something like
![image](https://github.com/user-attachments/assets/cd70b3b3-3f30-4b9e-9b43-b52ad5c52b2e)

Not sure on the UI yet but something like:

![image](https://github.com/user-attachments/assets/7d132893-a399-4abf-8cac-346d66fb9036)